src/components/app/DocumentList/index.js

```diff
@@ -424,7 +420,7 @@ class DocumentListContainer extends Component {
     filterView(windowType, viewId, filtersActive.toIndexedSeq().toArray())
       .then((response) => {
         const viewId = response.viewId;
-
+        this.connectWebSocket(viewId);
```

Comment:
Do we have to do this for all filters or barcode scanner is just some edge case ?

Reply:
For all filters. If you look at the URI when you setup new filter the viewId is changing